slemicro/README.md

```diff
@@ -183,6 +183,14 @@ worker-1   Ready    <none>                      15s   v1.25.9+k3s1
 
 **TIP:** To run the VM creation in parallel you can use `for file in vm-*; do ./create_vm.sh -f ${file} &; done; wait` (the output will be a little bit messy however)
 
+### Static IPs
+
+It is possible to deploy a VM with a static IP by setting the `VM_STATIC_IP` variable.
+
```

Comment:
I'd say to also include VM_NETMASK just in case.

Reply:
And maybe DNS?

Reply:
Sure - I was trying to keep it simple, but we can default them to sane values - I think in most cases DNS will be the same as the gateway (since DNS is provided via libvirt dnsmasq), but there may be cases where folks want to point somewhere else 👍

Reply:
added variables for gateway, prefix and DNS (all optional)